simulate_expression_compendia_modules/pipeline.py

```diff
@@ -0,0 +1,228 @@
+"""
+Author: Alexandra Lee
+Date Created: 11 March 2020
+
+Scripts called by analysis notebooks to run entire the entire analysis pipeline:
+1. Process data
+2. Run simulation experiment, described in `simulations.py`
+"""
+
+from simulate_expression_compendia_modules import simulations
+from ponyo import utils
+import os
+import pandas as pd
+import numpy as np
+import math
+
+from joblib import Parallel, delayed
+
+# import multiprocessing
+
+import warnings
+
+
+def fxn():
+    warnings.warn("deprecated", DeprecationWarning)
+
+
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore")
+    fxn()
+
+
+np.random.seed(123)
+
+
+def transpose_data(data_file, out_file):
```

Comment:
Is the point of this function to transpose the data and output it to a file? Is the data ever  used after this function is called? Based on the name I'd expected the function to return some sort of transformed data.

Reply:
This function is writing the transposed data to a file so that I don't need to call this function each time since so many files take expression data file as input and previously I was calling `.T` each time it was read in which made debugging difficult